curvefs/src/mds/chunkid_allocator.h

```diff
@@ -0,0 +1,146 @@
+/*
+ *  Copyright (c) 2020 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+/*
+ * Project: curve
+ * Created Date: 2021-08-27
+ * Author: chengyi01
+ */
+
+#ifndef CURVEFS_SRC_MDS_CHUNKID_ALLOCATOR_H_
+#define CURVEFS_SRC_MDS_CHUNKID_ALLOCATOR_H_
+
+#include <memory>
+#include <string>
+
+#include "src/common/concurrent/concurrent.h"
+#include "src/common/namespace_define.h"
+#include "src/common/string_util.h"
+#include "src/kvstorageclient/etcd_client.h"
+
+namespace curvefs {
+namespace mds {
+
+using ::curve::common::CHUNKSTOREKEY;
```

Comment:
do not use previous key, add another prefix key into https://github.com/opencurve/curve/blob/50fb805063da9bfb6bef3add0d6295412426ae7a/curvefs/src/mds/common/storage_key.h#L37, and in the same format.

Reply:
fix